handsontable/test/helpers/common.js

```diff
@@ -188,20 +189,19 @@ export function columnIndexMapper() {
  * @param {object} options The Handsontable options.
  * @param {boolean} explicitOptions If set to `true`, the options will be passed to the Handsontable instance as-is
  * and license key won't be added automatically.
+ * @param {jQuery} container The root element where the Handsontable will be injected.
  * @returns {Handsontable}
  */
-export function handsontable(options, explicitOptions = false) {
-  const currentSpec = spec();
-
+export function handsontable(options, explicitOptions = false, container = spec().$container) {
   // Add a license key to every Handsontable instance.
   if (options && !explicitOptions) {
     options.licenseKey = 'non-commercial-and-evaluation';
   }
 
-  currentSpec.$container.handsontable(options);
-  currentSpec.$container[0].focus(); // otherwise TextEditor tests do not pass in IE8
+  container.handsontable(options);
+  container[0].focus(); // otherwise TextEditor tests do not pass in IE8
```

Comment:
Maybe this line can be removed :)?

Reply:
Done within https://github.com/handsontable/handsontable/pull/10430/commits/90e205db77e39d699809a908093e7e9a54f398cd.

Reply:
Thank you ❤️